### PR TITLE
HOUSNAV-44: adding sections object to walkthrough 9.9.9

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
**Purpose**
Adding a top level object that denotes the sections of the walkthrough, their titles, and which questions are in each section. This will help construct the side navigation in the future.
I've also added tests to verify 2 things:
Each question that appears in a section actually exists
Each question either appears in a section or is of type "variable"

### CODE REVIEW

File: apps/web/components/walkthrough/Walkthrough.tsx [hunk 1]
```diff
@@ -37,12 +39,25 @@ export default function Walkthrough({
     // setVariable(currentQuestion.variableToSet);
```
Comment (jbook24) [inline apps/web/components/walkthrough/Walkthrough.tsx]: Do we want to remove this commented code?

**Review by jbook24 — APPROVED** [review]

File: apps/web/components/walkthrough/Walkthrough.tsx (hunk 1 above)
Comment (nicholaslyon-dt) [reply apps/web/components/walkthrough/Walkthrough.tsx]: Good call. Removed!

File: packages/data/walkthroughs/9.9.9.test.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          return false;
+        }
+
+        return !Object.values(data.sections).some((section) =>
+          section.sectionQuestions.includes(questionId),
+        );
+      },
+    );
+    if (possibleUnusedQuestion)
+      console.log("Possible unused question ID:", possibleUnusedQuestion[0]);
```
Comment (laurenrevere) [inline packages/data/walkthroughs/9.9.9.test.ts:22]: Do we need this console log in the test? Won't the expect tell us the value of `possibleUnusedQuestion` if the test fails?

**Review by wardm89 — APPROVED** [review]
LGTM; did some some manual tests to verify it updates correctly to the next question.

**Review by laurenrevere — APPROVED** [review]

File: packages/data/walkthroughs/9.9.9.test.ts (hunk 2 above)
Comment (nicholaslyon-dt) [reply packages/data/walkthroughs/9.9.9.test.ts:22]: It does, but it's not as easy to read/readily apparent IMO. This was just for readability.

**Pull request merged**